lib/beacon/router.ex

```diff
@@ -190,12 +190,33 @@ defmodule Beacon.Router do
       """
     end
 
+    on_mounts = get_on_mount_list(Keyword.get(opts, :on_mount))
```

Comment:
```suggestion
    on_mounts = get_on_mount_list(Keyword.get(opts, :on_mount, []))
```

and then you can remove both these clauses:

```elixir
  defp get_on_mount_list(nil), do: [BeaconWeb.Hooks.AssignAuthorizationContext]
  defp get_on_mount_list([]), do: [BeaconWeb.Hooks.AssignAuthorizationContext]
```